### PR TITLE
feat!: simplify alert components (#252)

### LINKED ISSUE #252: Simplify Alert component
- Use latest theme updates to simplify the alert component.
- Add size prop
- Remove variants
- Utilize color prop

### PR DESCRIPTION
**Pull request by frano-m**
Closes #252.

### CODE REVIEW

File: src/components/common/Alert/alert.styles.ts [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
- */
-/* eslint-enable valid-jsdoc -- disable require param */
-export const FluidAlert = styled(Alert)`
-  ${({ theme }) => theme.breakpoints.down(TABLET)} {
-    border-left: none;
-    border-radius: 0;
-    border-right: none;
-    box-shadow: none;
-  }
+export const StyledAlert = styled(Alert)`
```
Comment (frano-m) [inline src/components/common/Alert/alert.styles.ts:10]: The `Alert` component has been simplified and the only styles now relate to size: see [mock](https://www.figma.com/design/JOUDXkKfgivRNFdBsWDPEH/AnVIL?node-id=587-5515&t=zhf66W28cJJO6DxI-4) and [mock](https://www.figma.com/design/JOUDXkKfgivRNFdBsWDPEH/AnVIL?node-id=587-5534&t=zhf66W28cJJO6DxI-4).

File: src/theme/components/muiAlert.ts
Comment (frano-m) [inline src/theme/components/muiAlert.ts:1]: Simplifying alert theme; including removal of unwanted variants.

File: types/data-explorer-ui.d.ts [hunk 2]
```diff
@@ -25,21 +25,21 @@ import {
   CustomFeatureTableState,
 } from "../src/components/Table/features/entities";
 import { GridTrackSize } from "../src/config/entities";
+import { SIZE } from "../src/styles/common/constants/size";
 
 /**
  * Alert prop options.
  */
 declare module "@mui/material/Alert" {
+  interface AlertProps {
```
Comment (frano-m) [inline types/data-explorer-ui.d.ts:34]: Adding `size` to Mui Alert props.

**Review by MillenniumFalconMechanic — APPROVED** [review]
LGTM, thanks F!

**Pull request merged**